solidity/random-beacon/test/utils/dkg.ts

```diff
@@ -252,3 +267,24 @@ export async function signDkgResult(
 
   return { members, signingMembersIndices, signaturesBytes }
 }
+
+// Creates a members hash that actively participated in dkg
+export function hashDKGMembers(
+  members: number[],
+  misbehavedMembersIndices: number[]
+) {
```

Comment:
Let's fix `Missing return type on function` warning.
```suggestion
): string {
```

Reply:
449a226